src/react/containers/AppContainer.js

```diff
@@ -61,8 +62,7 @@ const mapStateToProps = state => ({
   page: state.pagination.page,
   loading: state.api.loading,
   entries: Object.keys(state.api.entries)
-    .map(key => state.api.entries[key])
-    .slice(0, state.config.entries_per_page),
```

Comment:
I had to remove this slice method, seems to still work perfectly. Would love some feedback on why this was used.